src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Vector;
+
+import Class.*;
+import ErrorHandling.*;
+
 public class Duke {
+    public static String lineDivider = "    --------------------------------------------------------------------------";
+
+    public static void addMessage(Task task, int size){
+        System.out.println("     Got it. I've added this task:");
+        System.out.println("     " + task.getDescription());
+        System.out.println("     Now you have " + size + " tasks in this list.");
+    }
+
+    public static void takeCommand(String command,List<Task> tasks) throws UnknownCommand, EmptyDescription {
```

Comment:
A long method here. Consider how details could be abstracted away, while maintaining [SLAP](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard).